client/blocks/inline-help/inline-help-center-content.scss

```diff
@@ -33,146 +25,222 @@
-	padding: 16px;
+			.search {
+				height: 40px;
+			}
+		}
 
-	.button {
-		width: 100%;
-	}
-}
+		.inline-help__results {
+			margin-bottom: 40px;
 
-.inline-help__richresult {
-	text-align: left;
+			.inline-help__results-item {
+				margin-bottom: 15px;
 
-	.inline-help__richresult__title {
-		display: block;
-		font-size: $font-body-small;
-		font-weight: 600;
-		margin-bottom: 5px;
-	}
-}
+				.inline-help__results-cell a {
+					display: flex;
 
-.inline-help__contact {
-	.inline-help__title {
-		@include hide-content-accessibly;
+					svg {
+						margin-right: 15px;
+						display: block;
+						padding: 5px;
+						background: var( --studio-gray-0 );
+						fill: var( --studio-blue-70 );
+					}
+
+					span {
+						display: block;
+						align-self: center;
+					}
+				}
+			}
+		}
 	}
 
-	.help-contact__form {
-		margin: 0;
-		padding: 0;
-		box-shadow: none;
-		text-align: left;
-		border-radius: 2px 2px 0 0;
+	/**
+	* RICH RESULTS
+	*/
+	.inline-help__secondary-view.inline-help__richresult {
+		button.button.is-borderless {
+			font-size: $font-body-small;
+			color: #000;
+		}
+	}
 
-		.sites-dropdown__wrapper {
-			max-width: 100%;
+	/**
+	* MORE RESOURCES
+	*/
+	.inline-help__more-resources {
+		.inline-help__resource-item {
+			margin-bottom: 15px;
+
+			.inline-help__resource-cell a {
+				display: flex;
+
+				svg {
+					margin-right: 15px;
+					display: block;
+					padding: 5px;
+					background: var( --studio-gray-0 );
+				}
 
-			.site__content {
-				padding: 8px;
+				span {
+					display: block;
+					align-self: center;
+				}
 			}
 		}
+	}
+
+	/**
+	* CONTACT FORM
+	*/
+	.inline-help__contact-page {
+		.inline-help__contact-boxes {
+			display: flex;
+			flex-direction: column;
+			justify-content: space-between;
+
+			.inline-help__contact-box {
+				display: flex;
+				flex-direction: row;
+				border: 1px solid var( --studio-gray-5 );
+				border-radius: 4px;
+				cursor: pointer;
+
+				&.chat {
+					margin-bottom: 15px;
+
+					.inline-help__contact-box-icon {
+						background-color: var( --color-accent );
+					}
+				}
+
+				// Text box
+				:not( .inline-help__contact-box-icon ) {
+					padding: 0 15px;
+					margin: 15px 0;
 
-		.help-contact-confirmation {
-			min-height: auto;
-			height: auto;
+					> * {
+						padding: 0;
+						margin: 0;
+					}
 
-			.help-contact-confirmation__contents {
-				margin: 20px 0;
+					p {
+						margin-top: 5px;
+					}
+
+					h2 {
+						font-weight: 400;
+					}
+				}
+
+				// Icon box
+				.inline-help__contact-box-icon {
+					display: flex;
+					background-color: var( --studio-blue-70 );
+					width: 60px;
+
+					svg {
+						fill: white;
+						display: block;
+						margin: auto;
+					}
+				}
 			}
 		}
-	}
 
-	.help-contact-form__site-selection {
-		margin-bottom: 8px;
-	}
+		.inline-help__results {
+			margin-top: 40px;
 
-	.sites-dropdown {
-		.site-selector__sites {
-			max-height: 20vh;
-		}
-	}
-}
+			> * {
+				margin: 0;
+			}
```

Comment:
This can stay but it's pretty hard to maintain these generic selectors, try to avoid them if you can. You can search for `inline-help__results` to find the affected element, but not for `*`.